src/main/java/seedu/address/logic/commands/schedule/DeleteScheduleCommand.java

```diff
@@ -20,8 +20,10 @@
     public static final String COMMAND_WORD = ScheduleCommand.COMMAND_WORD + " delete";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the interview identified by the index number in the interview schedule.\n"
-            + "Example: " + COMMAND_WORD + " 3";
+            + ": Deletes the interview identified by the index number in the displayed interview schedule.\n"
+            + "Parameters: INDEX\n"
+            + "Example: " + COMMAND_WORD + " 1\n"
```

Comment:
Same point as below - `INTERVIEW_INDEX` instead of `INDEX`.
Sorry ...

Reply:
yep got it! :)